eclipse-theia/theia
# Map vscode theme icons to codicons

#### What it does

Closes https://github.com/eclipse-theia/theia/issues/10746

Maps `ThemeIcon` IDs to their corresponding codicon. See [also here](https://code.visualstudio.com/api/references/icons-in-labels#icon-listing).

#### How to test

1. Install the [tree-view-sample](https://github.com/msujew/vscode-extension-samples/tree/msujew/vscode-theme-icons) extension ([download](https://github.com/msujew/vscode-extension-samples/raw/msujew/vscode-theme-icons/tree-view-sample/custom-view-samples-0.0.1.vsix))
2. Open a JSON file (such as the `package.json`) and open the `JSON Outline` view
3. Each node should have a random codicon assigned (none are displayed on `master`)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
